packages/calcite-components/src/components/input-message/input-message.scss

```diff
@@ -3,34 +3,39 @@
  *
  * These properties can be overridden using the component's tag as selector.
  *
- * @prop --calcite-input-message-spacing-value: The top margin spacing above the component.
+ * @prop --calcite-input-message-spacing-value: [Deprecated] Use `--calcite-input-message-spacing`.The top margin spacing above the component.
+ * @prop --calcite-input-message-spacing: Specifies the top margin space above the component.
```

Comment:
I think either this prop name or description could be a bit better - the "top margin space" is a little strange. cc @geospatialem or @DitwanP for doc consistency q there.